src/main/java/org/broadinstitute/hellbender/utils/haplotype/SAMFileDestination.java

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.hellbender.utils.haplotype;
+
+import htsjdk.samtools.*;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+// TODO - should this use SAMFileGATKReadWriter ?
```

Comment:
i think using File is ok for now, we can change later once we move on from files